docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
+
+For the purpose of this section of our documentation, we will assume that each DAG is completely accurate, represents **all** relevant variables in a given causal network and that there are no unrepresented variables.
+
+Types of Pathways Within Directed Acyclic Graphs
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+A path in a DAG is defined as any route drawn through arrows that link two variables **regardless of the direction of the arrows**.
+
+Directed Paths
+~~~~~~~~~~~~~~
+
+Directed paths are those in which variables are connected through arrows that flow all flow in the same direction. See below for an example of a directed path between variables A and D.
+
+.. image:: directional_dag.svg
+
+**In the case of a directed path between two variables, those variables will be causally related.**
+
+Undirected Paths
+~~~~~~~~~~~~~~~~
+
+Undirected paths are those in which variables are connected through arrows that do not all flow from head to tail. The two cases of undirected paths covered here include backdoor paths and colliders.
+
+**In the case that there is an undirected path (and no directed path) bewteen two variables, those variables will NOT be causally related, although they may be associated in some way.**
+
+Backdoor Paths
+^^^^^^^^^^^^^^
+
+Backdoor paths are those in which two variables are connected via a common cause. In other words, two variables that are linked via a variable that has two arrows flowing *away* from it. See below for an example in which variables Y and Z are connected via a backdoor path through variable X.
+
+.. image:: backdoor_path_dag.svg
+
+**In the case of a backdoor path between two variables (and no directional path), there will be an association (due to a common cause between them), but no causal relationship between the two variables.**
+
+Colliders
+^^^^^^^^^
+
+Colliders are variables that have more than one cause; in other words, where pathways collide and a variable has two arrows flowing *toward* it. See below for an example in which variable Z is a collider on the path between X and Y.
+
+.. image:: collider_dag.svg
+
+**In the case that two variables are linked through a pathway with a collider (and there is no directional or backdoor pathway between them), there will be no association or causal realtionship between the two variables.** This is also referred to as a **"closed path."**
+
+Summary
+~~~~~~~
+
+.. list-table:: DAG Pathways
+   :widths: 5 5 5 5 5
+   :header-rows: 1
+
+   * - Path Name/Term
+     - Description
+     - Direction
+     - Open or closed
+     - Association
+   * - Direct path
```

Comment:
Is this a 'direct' or a 'directed' path, or are both terms used? if it's one or the other, i think it would be helpful to make this chart align with the section from line 711-714!

Reply:
This should be "Directed path".